packages/react-composites/src/composites/localization/locales/es-ES/strings.json

```diff
@@ -324,11 +324,18 @@
       "stopSpotlightCancelButtonLabel": "Cancelar"
     },
     "exitSpotlightButtonLabel": "Salir del primer plano",
-    "exitSpotlightButtonTooltip": "Salir del primer plano"
+    "exitSpotlightButtonTooltip": "Salir del primer plano",
+    "hangUpConfirmButtonLabel": "Confirmar",
+    "hangUpCancelButtonLabel": "Cancelar",
+    "leaveConfirmDialogTitle": "¿Abandonar la llamada?",
+    "leaveConfirmDialogContent": "¿Está seguro de que desea abandonar la llamada?",
+    "endCallConfirmDialogTitle": "¿Finalizar la llamada?",
+    "endCallConfirmDialogContent": "Finalizará la llamada para todos los usuarios.",
+    "invalidMeetingIdentifier": "Id. de reunión o código de acceso no válidos"
   },
   "chat": {
     "chatListHeader": "En este chat",
-    "uploadAttachment": "Subir archivo"
+    "uploadAttachment": "Upload Attachment"

```

Comment:
Okay, not gonna comment about this one anymore cause looks to be missing on a lot of them at the moment. Might need to wait a bit more until this gets translated

Reply:
![image](https://github.com/Azure/communication-ui-library/assets/94866715/e2b3aba3-97dd-4f8c-9d7d-2f9f93ca1be8)
looks like refetching fixed the issue